models/email_request.go

```diff
@@ -96,7 +96,12 @@ func (s *SendTestEmailRequest) Generate(msg *gomail.Message) error {
 	if err != nil {
 		Logger.Println(err)
 	}
-	msg.SetHeader("Subject", subject)
+	// don't set the Subject header if it is blank
+	if len(subject) == 0 {
+		Logger.Println("Subject is empty and the Subject header was not set")
```

Comment:
I'd say we don't have to worry about logging this for now. In the future, I might have some validation like you'll find in email clients where maybe before the user saves the email template it'll validate if they want to have an empty subject.

Reply:
Hey,

Thanks for your feedback. Will make the changes. My original plan was to have a checkbox in Email template which would say “don’t send subject” that would grey out the field. However, that would mean differentiating between the case where a blank subject should be sent and where none should be sent at all. That would mean schema change unless the subject would have some magic marker like —GOPHISH-DONTSENDSUBJECT— which felt hacky and I didn’t feel like doing a schema change.

Reply:
Hi, quick question: how do I run the test suite, for some reason go test wasn't helpful:

Running tool: C:\Go\bin\go.exe test -timeout 30s github.com\gophish\gophish\models -run ^$

ok  	github.com/gophish/gophish/models	0.110s **[no tests to run]**
Success: Tests passed.

Reply:
I wouldn't worry about the checkbox - it seems much more usable to just alert when the subject is blank and even going so far as to alert is debatable - we might as well as just trust the user to make the choices they want to make. Something we'll add on later if we get complaints 😄 

To run the test suite, try going into the models directory and running `go test -v ./...`. It usually works pretty well.

Let me know if you run into any issues! Thanks again for all your work on this.